lib/SPIRV/SPIRVReader.cpp

```diff
@@ -3528,11 +3529,14 @@ void SPIRVToLLVM::transIntelFPGADecorations(SPIRVValue *BV, Value *V) {
   }
 }
 
-void SPIRVToLLVM::transUserSemantic(unsigned FunNum) {
-  auto Fun = BM->getFunction(FunNum);
+// Having UserSemantic decoration on Function is against the spec, but we allow
+// this for various purposes (like prototyping new features when we need to
+// attach some information on function and propagate that through SPIR-V and
+// ect.)
+void SPIRVToLLVM::transUserSemantic(SPIRV::SPIRVFunction *Fun) {
   auto transFun = transFunction(Fun);
```

Comment:
```suggestion
  auto TransFun = transFunction(Fun);
```

According to [LLVM Coding Standards](https://llvm.org/docs/CodingStandards.html#name-types-functions-variables-and-enumerators-properly), variable names should start with uppercase letter